python/pyspark/ml/classification.py

```diff
@@ -380,7 +380,7 @@ def areaUnderROC(self):
     @since("2.0.0")
     def pr(self):
         """
-        Returns the precision-recall curve, which is an Dataframe
+        Returns the precision-recall curve, which is a Dataframe
```

Comment:
I think a lot of these problem came when someone searched-and-replaced "RDD" for "DataFrame" but of course it was "an RDD" before. OK.